homeassistant/components/heos/media_player.py

```diff
@@ -37,6 +57,9 @@ class HeosMediaPlayer(MediaPlayerDevice):
     def __init__(self, player):
         """Initialize."""
         self._player = player
+        self._position_jitter_acceptance_ms = 500
+        self._cache_position_ms = 0
+        self._cache_position_at = datetime.now(UTC)
```

Comment:
Use our dt.py util as possible. In this case `utcnow`.

Reply:
fixed